### PR TITLE
fix(web): add unique/required/multivalue options

### PR DESCRIPTION
**Pull request by issmail-basel**
# Overview
Update mutation for fields wasn't available, added needed queries/hooks/component changes to reflect backend changes

### CODE REVIEW

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx [hunk 1]
```diff
@@ -235,12 +244,25 @@ const FieldUpdateModal: React.FC<Props> = ({
...
+            <Form.Item
+              name="required"
+              valuePropName="checked"
+              extra={t("Prevents saving an entry if this field is empty")}>
+              <Checkbox>{t("Make field required")}</Checkbox>
+            </Form.Item>
+            <Form.Item
+              name="unique"
+              valuePropName="checked"
+              extra={t("Ensures that a multiple entries can't have the same value for this field")}>
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx:263]: Ensures that multiple entries can't have the same value for this field

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx [hunk 2]
```diff
@@ -235,12 +244,25 @@ const FieldUpdateModal: React.FC<Props> = ({
...
               extra={t("Stores a list of values instead of a single value")}>
               <Checkbox>{t("Support multiple values")}</Checkbox>
             </Form.Item>
           </TabPane>
           <TabPane tab={t("Validation")} key="validation" forceRender>
             <FieldValidationInputs selectedType={selectedType} />
+            <Form.Item
+              name="required"
+              valuePropName="checked"
+              extra={t("Prevents saving an entry if this field is empty")}>
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx:257]: Prevent saving an entry if this field is empty

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx (hunk 1 above)
Comment (issmail-basel) [reply web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx]: @KaWaite, updated.

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx (hunk 2 above)
Comment (issmail-basel) [reply web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx]: @KaWaite, updated.

Comment (KaWaite) [reply web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx]: Sorry my bad. I am mixing writing styles. Can you revert this? The description for Unique is `Ensures that....`, so here should be `Prevents saving...`.
My bad. Apologies

Comment (issmail-basel) [reply web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx:257]: @KaWaite , updated.

**Review by KaWaite — APPROVED** [review]

**Pull request merged**